### ISSUE TITLE
[RAI] November Release: Engagement and Becoming Engaged Again

### ISSUE THREAD

**Issue by lenaleciel** [AshleyMoni/Lancer-Community-Edition#634]
**The Problem:**
The text is unclear as to whether or not moving to a different space adjacent to a hostile character one is already engaged with will cause you to stop moving immediately and lose the rest of your movement.
Currently, engagement is described as "If a character moves adjacent to a hostile character, they both gain the ENGAGED status for as long as they remain adjacent to one another." Also relevant to this is the fact that "Additionally, characters that become ENGAGED by targets of equal or greater SIZE during the course of a movement stop moving immediately and lose any unused movement."
In other words, can you become Engaged by someone you are already Engaged with, thus stopping further movement?

**The Solution:**
Clarify whether or not you are supposed to stop and lose all your movement if you step into a different space that's adjacent to an enemy you are already Engaged with. In other words, say that you either *can* or *cannot* become Engaged again by an enemy you are Engaged with.

**The Location:**
pg 62 Engagement

**Comment by lenaleciel** [comment]
Engagement now also reads "Engagement does not stop you from movingWhile
engaged with a character, you can move normally as long as you don’t break engagement with that character."
...This is kind of weird, as it implies that if you try to get away, you can't move at all, as opposed to "if you try, you'll be punished by overwatch."
